docs/projects/fps/tutorial.md

```diff
@@ -259,11 +267,11 @@ The FPS Starter Project contains a health pack prefab named `HealthPickup.prefab
 
 This is because the FPS Starter Project uses the GDK GameObject Creation package as part of the MonoBehaviour workflow. To find out more you can read up about the [MonoBehaviour workflow]({{urlRoot}}/content/intro-workflows-spatialos-entities).<%(/Expandable)%>
 
-<%(#Expandable title="What's the best way to create a prefab?")%>Prefabs are the Unity Engine approach for creating templates of GameObject hierarchies.
+<%(#Expandable title="What's the best way to create a prefab?")%>Prefabs are the Unity Engine approach for creating templates of GameObject hierarchies. [Creating Prefabs (Unity Documentation)](https://docs.unity3d.com/Manual/CreatingPrefabs.html) explains how to create one.
```

Comment:
We can safely ditch the "(Unity Documentation)" addition going forward, if we want to - we now have a lovely icon. Can leave it in if we want.

Reply:
No need to ditch it here - but just while I remember.